roman.py

```diff
@@ -1,13 +1,17 @@
+import re
+
 def roman_to_integer(value):
```

Comment:
What's happens if I pass a invalid value? Like 'VX' or 1234?
We should have test cases for those scenarios.

Reply:
It will fail to validate those, yes, more scenarios are needed to cover those cases.